templates/vue/cypress/integration/tyde.spec.js

```diff
@@ -0,0 +1,318 @@
+describe("TYDE", () => {
+  const cockpitBackgroundImage =
+    "https://m.media-amazon.com/images/I/51h5kRXyipL._AC_.jpg"
+
+  const icons = {
+    notEarned:
+      "https://cdn.imgbin.com/11/2/4/imgbin-cyanide-happiness-satire-iran" +
+      "-cyanide-and-happiness-PRxa77GZR5WSHJzh85PDGEESy.jpg",
+    earned:
+      "https://img.favpng.com/13/0/15/cyanide-happiness-character-drawing" +
+      "-png-favpng-pnWdvwDziWgdHuQjEjgtWjXEv.jpg",
+    cockpit: {
+      notEarned:
+        "https://lh3.googleusercontent.com/proxy/EfGXb_q2dzJt5VBgQuXuNjJG" +
+        "wLWO0tlSdVz5gTalgRNzkiso3HtplzRhGOpyYsaS4-4kZuda4SUewK0UcnhJPfYm" +
+        "uDzIp1Ohyg",
+      earned:
+        "https://www.clipartmax.com/png/middle/29-292956_world-globe-clip" +
+        "-art-clipart-2-wikiclipart-wikiclipart-transparent-background-ea" +
+        "rth-clipart.png",
+      hover:
+        "https://www.clipartmax.com/png/middle/29-292956_world-globe-clip" +
+        "-art-clipart-2-wikiclipart-wikiclipart-transparent-background-ea" +
+        "rth-clipart.png",
+    },
```

Comment:
Could we use some images that are from a more reliable source like Wikimedia?

Reply:
Good call